tests/eval-compare/builtins.pathExists.nix

```diff
@@ -0,0 +1 @@
+builtins.pathExists "/bin/sh/invalid-director"
```

Comment:
Does this path exist on all machines?

Reply:
No, I saw the previous test assert that a path does not exist.